frontend/.eslintrc.js

```diff
@@ -19,5 +19,8 @@ module.exports = {
     sourceType: "module",
   },
   plugins: ["react", "@typescript-eslint"],
-  rules: {},
+  rules: {
+    quotes: ["error", "double"],
+    indent: ["error", 4],
```

Comment:
@alincustura is this rule erroring when there are 4 spaces or is it ensuring there are 4 spaces? I'm not to familiar with how eslint works in this respect. Just make sure it's 2 spaces indenting, to match prettier

Reply:
ok

Reply:
Ok